core/chainlink.goreleaser.Dockerfile

```diff
@@ -20,8 +20,9 @@ COPY ./chainlink /usr/local/bin/
 # Copy native libs if cgo is enabled
 COPY ./tmp/linux_${TARGETARCH}/libs /usr/local/bin/libs
 
-# Copy plugins and enable them
-COPY ./tmp/linux_${TARGETARCH}/plugins/* /usr/local/bin/
+# Copy plugins if exist and enable them
+# https://stackoverflow.com/questions/70096208/dockerfile-copy-folder-if-it-exists-conditional-copy/70096420#70096420
+COPY ./tmp/linux_${TARGETARCH}/plugin[s] /usr/local/bin/
```

Comment:
Based on that stackoverflow this should have a trailing `/`?

ie. `./tmp/linux_${TARGETARCH}/plugin[s]/`

Reply:
ahh good catch!

Reply:
discussed offline 👍